seq_tools/validation/c120_permissible_char_in_rg_id.py

```diff
@@ -47,7 +47,7 @@ def check(self):
         resp=requests.get(url)
         
         if resp.status_code==200:
-            regex=resp.json()['schema']['properties']['read_groups']['items']['allOf'][0]['properties']['submitter_read_group_id']['pattern']
+            regex=resp.json()['schema']['properties']['read_groups']['items']['properties']['submitter_read_group_id']['pattern']
         else:
             regex='^[a-zA-Z0-9\\-_:\\.]+$'
```

Comment:
@edsu7 , can you also update the regex to the same as what is defined in `pattern`?